code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -113,14 +113,143 @@
 
 /obj/item/nullrod/godhand
 	name = "god hand"
-	icon_state = "disintegrate"
-	item_state = "disintegrate"
-	desc = "This hand of yours glows with an awesome power!"
+	icon_state = "burnhand"
+	item_state = "burnhand"
+	desc = "Like Prometheus, you steal fire from the Gods! Except you didn't steal this."
 	flags = ABSTRACT | NODROP| DROPDEL
 	w_class = WEIGHT_CLASS_HUGE
 	hitsound = 'sound/weapons/sear.ogg'
+	force = 15
```

Comment:
This is unneeded, because its parent (/obj/item/nullrod) already has a force of 15.